api/src/main/java/edu/cornell/mannlib/vitro/webapp/audit/AuditResults.java

```diff
@@ -0,0 +1,39 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+
+package edu.cornell.mannlib.vitro.webapp.audit;
+
+import java.util.List;
+
+/**
+ * Results object for retrieving entries from the audit store
+ */
+public class AuditResults {
+    private long total;
+    private long offset;
+    private long limit;
+
+    private List<AuditChangeSet> datasets;
+
```

Comment:
Can you provide a private default constructor?

Reply:
Why do we need a private default constructor here?

Reply:
Good practice. But I have tested with https://www.jdoodle.com/online-java-compiler/ and confirmed that the default constructor is not able to be called outside of reflection or classpath overlay when a constructor is declared.